content/php-client/ee-resources/asset-variation-files.md

```diff
@@ -0,0 +1,79 @@
+## Asset variation file
+
+### Get a variation file of a localizable asset
+
+```php
+$client = new \Akeneo\Pim\ApiClient\AkeneoPimClientBuilder('http://akeneo.com/')->buildAuthenticatedByPassword('client_id', 'secret', 'admin', 'admin');
+
+/*
+ * Returns an array like this:
+ * [
+ *     'code'    => 'f/5/5/c/f55c7ea4adae17d4e02f4d04a839bc2a7cdbf165_chicago_skyline_en_US_mobile.jpg',
+ *     'locale'  => 'en_US',
+ *     'channel' => 'mobile',
+ *     '_link'   => [
+ *         'download' => [
+ *             'href' => 'http://akeneo.com/api/rest/v1/assets/chicagoskyline/variation-files/mobile/en_US/download',
+ *         ],
+ *     ],
+ * ]
+ */
+$product = $client->getAssetVariationFileApi()->getFromLocalizableAsset('chicagoskyline', 'mobile', 'en_US');
+```
+
+### Get a variation file of a not localizable asset
+
+```php
+$client = new \Akeneo\Pim\ApiClient\AkeneoPimClientBuilder('http://akeneo.com/')->buildAuthenticatedByPassword('client_id', 'secret', 'admin', 'admin');
+
+/*
+ * Returns an array like this:
+ * [
+ *     'code'    => '0/7/2/1/07217eea32563821b46336d2dec696e4f69415ec_bridge_mobile.jpg',
+ *     'locale'  => null,
+ *     'channel' => 'mobile',
+ *     '_link'   => [
+ *         'download' => [
+ *             'href' => 'http://akeneo.com/api/rest/v1/assets/bridge/variation-files/mobile/no-locale/download',
+ *         ],
+ *     ],
+ * ]
+ */
+$product = $client->getAssetVariationFileApi()->getFromNotLocalizableAsset('bridge', 'mobile');
```

Comment:
Move to line 28 pls

Reply:
Do I change that for all the other end-points too ? @NolwennP

Reply:
Nope, it's good this way.